src/commands/synthetics/run-test.ts

```diff
@@ -349,6 +355,7 @@ export class RunTestCommand extends Command {
         apiKey: process.env.DATADOG_API_KEY,
         appKey: process.env.DATADOG_APP_KEY,
         datadogSite: process.env.DATADOG_SITE,
+        locations: process.env.DATADOG_LOCATIONS?.split(';'),
```

Comment:
Could you rename it to `DATADOG_SYNTHETICS_LOCATIONS` as this environment variable is specific to the synthetics command?